<launch>
    <include file="$(find joybro)/launch/joy_serial.launch"/>

    <node pkg="joybro" type="dog_teleop.py" name="dog_teleop" output="screen"> 
        <param name="max_linear_vel" value="0.5" type="double"/>
        <param name="max_angular_vel" value="2.0" type="double"/>
        <param name="threshold" value="40" type="double"/>
    </node>
</launch>